tests/lib/azure/vnet.go

```diff
@@ -24,13 +25,21 @@ func NewVnet(f fixture.F) *Vnet {
 
 // AssertExists checks if virtual network exists in the resource group.
 // Fail tests otherwise.
-func (vnet *Vnet) AssertExists(t *testing.T, name string, address string) {
+func (vnet *Vnet) AssertExists(t *testing.T, name string, address, routeTable string) {
```

Comment:
Thanks for checking that, I agree with you, it probably went unnoticed by myself :)